app/models/payout.rb

```diff
@@ -62,5 +62,7 @@ def nonprofit_must_be_vetted
 		self.errors.add(:nonprofit, "must be vetted") unless self.nonprofit && self.nonprofit.vetted 
 	end
 
+	def self.publish_created
```

Comment:
`#publish_created` should be an instance method. The event is based upon the characteristics of a specific Payout.

Reply:
I see what you're saying. I'm not sure that payouts are associated with object events, though (`example_payout.object_events` throws an error). Would that be something we should change?

Reply:
It is! I forgot it and it's in #600`